src/java/fr/paris/lutece/plugins/forms/web/FormXPage.java

```diff
@@ -361,7 +361,7 @@ public synchronized XPage getStepView( HttpServletRequest request ) throws SiteM
                 SiteMessageService.setMessage( request, FormsConstants.MESSAGE_ERROR_INACTIVE_FORM, SiteMessage.TYPE_ERROR );
             }
         }
-        IsRequestComingFromAction = false;
+        IsRequestComingFromAction = true;
```

Comment:
The original error from the issue seems related to another plugin (breadcrumbaccordion), it seems dangerous to change this behaviour, because if this boolean is set to true, it never will return to false ?

Reply:
The error related to breadcrumbaccordion is fixed [here](https://github.com/lutece-platform/lutece-form-module-forms-breadcrumbaccordion/pull/1)

Reply:
linked to [this PR](https://github.com/lutece-platform/lutece-form-plugin-forms/pull/300/commits)